### PR TITLE
Move-to-ubuntu-22.04

### PR DESCRIPTION
**Pull request by jdroenner**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by jdroenner** [comment]
[Update zu Ubuntu 22.04.](https://trello.com/c/71kHsnAD/1045-update-zu-ubuntu-2204)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -111,15 +111,15 @@ jobs:
...
         features: ["", "--features pro", "--all-features"]
         build: ["", "--release"]
 
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-      - name: Add UbuntuGIS ppa
-        run: sudo add-apt-repository ppa:ubuntugis/ppa
+      # - name: Add UbuntuGIS ppa
+      #  run: sudo add-apt-repository ppa:ubuntugis/ppa
```
Comment (ChristianBeilschmidt) [inline .github/workflows/ci.yml:122]: Which GDAL version is used here?

I guess you need to remove it if this is no necessary anymore.
Moreover, you have to adapt the readme.

File: .github/workflows/coverage.yml [hunk 2]
```diff
@@ -29,8 +29,8 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-      - name: Add UbuntuGIS ppa
-        run: sudo add-apt-repository ppa:ubuntugis/ppa
+      # - name: Add UbuntuGIS ppa
+      #  run: sudo add-apt-repository ppa:ubuntugis/ppa
```
Comment (ChristianBeilschmidt) [inline .github/workflows/coverage.yml:33]:
```suggestion
```

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]
Please merge the current master

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jdroenner) [reply .github/workflows/ci.yml:122]: Ubuntu 22.04 ships 3.4.1 as you can see here: https://packages.ubuntu.com/jammy/gdal-bin
Maybe we should gp kach to ubuntu gis at a later point. Ubuntu-gis unstable ships 3.4.3 as you can see here: https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable?field.series_filter=jammy
There is no ubuntu gis stable for 22.04 (yet).

Comment (jdroenner) [reply .github/workflows/ci.yml]: [7674a71](https://github.com/geo-engine/geoengine/pull/648/commits/7674a71878fa2c7964cd2e3e8d13ef6ba12bdb7b)

File: .github/workflows/coverage.yml (hunk 2 above)
Comment (jdroenner) [reply .github/workflows/coverage.yml]: [2472e0c](https://github.com/geo-engine/geoengine/pull/648/commits/2472e0c59fedf671dfb3f7ed21b589de9d873765)

File: README.md [hunk 3]
```diff
@@ -12,28 +12,17 @@ This workspace contains the Geo Engine crates.
...
-
 ### Dependencies
 
 ```
 # Build essentials
 apt install build-essential
 # lld linker
 apt install clang lld
-# GDAL (>= 3.2.1)
+# GDAL (>= 3.2.1), Ubuntu 22.04 ships GDAL 3.4.1
```
Comment (ChristianBeilschmidt) [inline README.md:25]: Just expect >= 3.4 ?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (jdroenner) [reply README.md]: [0b8eecd](https://github.com/geo-engine/geoengine/pull/648/commits/0b8eecdf42e867390589ab2bace4dca5568c76a8)

**Review by ChristianBeilschmidt — APPROVED** [review]

File: .github/workflows/coverage.yml [hunk 4]
```diff
@@ -53,7 +51,7 @@ jobs:
           # An additional cache key that is added alongside the automatic `job`-based
           # cache key and can be used to further differentiate jobs.
           # Default: empty
-          key: ""
+          key: "1"
```
Comment (ChristianBeilschmidt) [inline .github/workflows/coverage.yml:54]: Kannst du hier einen Namen geben und nicht mehr hochzählen?
Es gibt ja jetzt eine Action zum Leeren des Caches.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: .github/workflows/coverage.yml (hunk 4 above)
Comment (jdroenner) [reply .github/workflows/coverage.yml]: [f34b014](https://github.com/geo-engine/geoengine/pull/648/commits/f34b014db2b0770e3b041e6394dca4a5aca56411)

**Review by ChristianBeilschmidt — DISMISSED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**